src/SemanticUI/Modules/Dropdown/Select.elm

```diff
@@ -74,6 +75,9 @@ variationClassList variation =
         Selection { compact } ->
             [ ( "compact", compact ), ( "selection", True ) ]
 
+        MenuItem ->
+            [ ( "inline", True ), ( "item", True ) ]
```

Comment:
I don't see `inline` in the semantic ui examples for this - possible it's specific to the menu you're building?